src/index.html

```diff
@@ -5,9 +5,56 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+      <div class="star star--1"></div>
+      <div class="star star--2"></div>
+      <div class="star star--3"></div>
+      <div class="star star--4"></div>
+      <div class="star star--5" ></div>
```

Comment:
I think these modifiers are unnecessary 
You can add some styles using the block modifier `stars--0`

```suggestion
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star" ></div>
```